crates/rolldown_plugin/src/plugin_driver/build_hooks.rs

```diff
@@ -9,14 +9,19 @@ impl PluginDriver {
   pub async fn build_start(&self) -> HookNoopReturn {
     // TODO should call `build_start` of all plugins in parallel
     for (plugin, ctx) in &self.plugins {
-      plugin.build_start(ctx).await?;
```

Comment:
Said above, these changes to `PluginDriver` is unnecessary. They should be hide into the `ThreadSafePlugin`.